ACD-618 (#30)
Actindo-AG/Documentation: Channels/Integration/01_ManageConnections.md

[!!User interface Connections](../UserInterface/to_be_completed)

# Manage the connections

Establish connections via drivers to connect your marketplaces with the *Actindo Core1 Platform*. A marketplace can be either an online shop or a point of sale. You can establish an unlimited number of connections per driver.

> 

Reviewer: "You can establish an unlimited number of connections per driver." 
Stimmt/Macht das so Sinn? In Fulfillment ist es so gedacht, laut Stefan, dass man 1:1 verbindet, also 1 Driver - 1 Connection - 1 Fufiller.

> In the following, the general settings to create, edit, synchronize or disable connections are described. For certain connection drivers, you have to setup special configurations. Therefore, the connections below are described in separate documents:

Reviewer: ```suggestion
In the following, the general settings to create, edit, synchronize or disable connections are described. For certain connection drivers, you have to set up special configurations. Therefore, the connections below are described in separate documents:
```

[comment]: <> (POS auch in eigener Procedure erklären?)

#### Prerequisites

At least one driver has been installed.

#### Procedure

*Omni-Channel > Settings > Tab CONNECTIONS*

![Connection](../../Assets/Screenshots/Channels/Settings/Connections/Connection.png "[Connection]")

1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.    
  The *Create connection* view is displayed.

  ![Create connection](../../Assets/Screenshots/Channels/Settings/Connections/CreateConnection.png "[Create connection]")

2. Enter a name for the connection in the *Name* field.

3. Click the *Driver* drop-down list and select the desired driver. All available drivers are displayed in the list.   

  > [Info] Drivers must be purchased from the main account and then be installed separately in each sandbox or account where you want to use it.    
  For detailed information, see [Manage the drivers](to_be_completed).

>   Depending on the selected driver, a *Credentials* section is displayed below the drop-down list.

Reviewer: ```suggestion
  The *Credentials* section is displayed below the drop-down list.
```
Es kommt immer eine Credentials sections vor, oder?

  ![Credentials](../../Assets/Screenshots/Channels/Settings/Connections/Credentials.png "[Credentials]")

  > [Info] Depending on the driver of the selection, the fields in the *Credentials* section differ.

> 4. If necessary, enter the required data in the fields in the *Credentials* section.

Reviewer: ```suggestion
4. Enter the required data in the fields in the *Credentials* section.
```
Ich gehe davon aus, dass man immer die Loggin-Details eintragen muss.
